### ISSUE TITLE
Mobileclient.search(): no song_hits entries for tracks I have uploaded

### ISSUE THREAD

**Issue by darryllee** [simon-weber/gmusicapi#625]
Hi, not sure if this is related to #622 . I'm trying to use geemusic to play songs that I've manually uploaded to Google Music (not available on Google Play Music, even if I pay).

Doing a search for "me myself and i de la soul" (song + artist), I get 'song_hits': [].

(Interestingly, and annoyingly, I get zero Song results when I do this same search on the iOS app, so maybe it's just Google?)

When I search for "me myself and i" I get 1 hit, but it's not the one I uploaded, which *does* show up when I do Mobileclient.get_all_songs.

(In this case, iOS app *does* show the song I'm looking for as the first result.)

Is this expected behavior (Search does not look in uploaded songs?)

Should I consider modifying geemusic to just use get_all_songs as a local searchable index of my songs, since that's all I'm really concerned with playing?

Thanks!

**Comment by simon-weber** [comment]
Right, Library search seems to be done clientside and then mixed into the results from this endpoint.
